agent/go.sum

```diff
@@ -744,6 +744,7 @@ github.com/rs/xid v1.2.1/go.mod h1:+uKXf+4Djp6Md1KODXJxgGQPKngRmWyn10oCKFzNHOQ=
 github.com/rs/zerolog v1.13.0/go.mod h1:YbFCdg8HfsridGWAh22vktObvhZbQsZXe4/zB0OKkWU=
 github.com/rs/zerolog v1.15.0/go.mod h1:xYTKnLHcpfU2225ny5qZjxnj9NvkumZYjJHlAThCjNc=
 github.com/russross/blackfriday/v2 v2.0.1/go.mod h1:+Rmxgy9KzJVeS9/2gXHxylqXiyQDYRxCVz55jmeOWTM=
+github.com/ryanbressler/CloudForest v0.0.0-20161201194407-d014dc32840a/go.mod h1:295JVys1fl5dBchZafEH0L0FfJONcP/6lnXweezSn5w=
```

Comment:
Not quite sure what to do about this. The `go install` of growforest in the Makefile seems to be what triggers this. That seems... wrong. `go mod tidy` gets rid of it, but this is going to keep showing up every time someone else builds if I can't suppress it.

Reply:
Ooohhhh it's the GO111MODULE thing. The go tooling will detect it's running inside a module, so that's how it's ending up in there even though the code never references it (except through the process invocation - but I tested - that has nothing to do with it). Just turning that off when doing the `go install` fixes it.